includes/class-wcal-common.php

```diff
@@ -184,7 +184,8 @@ private static function wcal_get_recovered_amount( $wcal_data ) {
 
 		foreach ( $wcal_data as $wcal_data_key => $wcal_data_value ) {
 
-			$wcal_order_total             = get_post_meta( $wcal_data_value->recovered_cart, '_order_total', true );
+			$order                        = wc_get_order( $wcal_data_value->recovered_cart );
+			$wcal_order_total             = $order->get_total();
```

Comment:
Please modify it as `$wcal_order_total             = $order ? $order->get_total() : 0;`